pkg/util/template/engine.go

```diff
@@ -27,6 +28,23 @@ type Engine struct {
 	Resolver TemplateResolver
 }
 
+func (e *Engine) RenderTranslation(ctx *RenderContext, typ string, key string, data interface{}) (string, error) {
+	translations, err := e.Resolver.ResolveTranslations(
+		&ResolveContext{PreferredLanguageTags: ctx.PreferredLanguageTags},
+		typ,
+	)
+	if err != nil {
+		return "", err
+	}
+
+	return e.renderText(ctx, &Resolved{
```

Comment:
One concern here is the developer may want to include HTML in the translation (e.g. HTML email)